.gitignore

```diff
@@ -13,3 +13,16 @@ yarn.log
 
 # Don't commit ESLint error reports
 eslint_report.htm
+
+# Created by https://www.gitignore.io/api/visualstudiocode
```

Comment:
I would change that simply to "Taken from" because semantically it makes more sense without removing the attribution to gitignore.io!